File: 016/Problem16.mjs
import { performance } from "perf_hooks";

function problem() {
  var two = BigInt(2n);
  var thousand = BigInt(1000n);
  var number = two ** thousand;
  var numStr = number.toString();
  var sum = 0;
  for (var i = 0; i < numStr.length; i++) {
    sum += parseInt(numStr[i]);
  }
  console.log(sum);
}

var start = performance.now();
problem();
var end = performance.now();
console.log("Function took " + (end - start) / 1000);

File: 016/Problem16.cs
using System;
using System.Numerics;

namespace EulerProblems
{
    class Program
    {
        static void Main(string[] args)
        {
            var watch = new System.Diagnostics.Stopwatch();

            watch.Start();
            Problem();
            watch.Stop();

            Console.WriteLine($"Function Took: {watch.Elapsed}");
        }

        private static void Problem()
        {
            var two = new BigInteger(2);
            var sum = 0;
            var number = BigInteger.Pow(two, 1000);
            var numStr = number.ToString();
            for (var i = 0; i < numStr.Length; i++)
            {
                sum += int.Parse(Convert.ToString(numStr[i]));
            }
            Console.WriteLine(sum);
        }
    }
}